### PR TITLE
ver 2.4

### PR DESCRIPTION
**Pull request by shahriariravanian**
import functionality expanded
structures added

### CODE REVIEW

**Comment by anandijain** [comment]
🎉

File: src/analysis.jl [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    A collection of analytic tools to evaluate CellML models
+"""
+
+function simplify_systems(systems)
+    for x in systems
+        sys = last(x)
+        print("simplifying $(sys.name)")
+        try
+            sys = structural_simplify(sys)
+            k = max(1, 50-length(string(sys.name)))
```
Comment (anandijain) [inline src/analysis.jl:11]: ?

File: src/analysis.jl [hunk 2]
```diff
@@ -0,0 +1,19 @@
+"""
+    A collection of analytic tools to evaluate CellML models
+"""
+
+function simplify_systems(systems)
+    for x in systems
+        sys = last(x)
+        print("simplifying $(sys.name)")
```
Comment (anandijain) [inline src/analysis.jl:8]: actually eventually lets make stdout stuff a verbose flag

Comment (shahriariravanian) [reply src/analysis.jl:8]: I agree. However, this function is a debug function that is called manually. It is not called by any other function.

**Pull request merged**